input/fsh/CancerRelatedMedicationStatement.fsh

```diff
@@ -16,10 +18,15 @@ MedicationStatement resources associated with an [mCODE patient](conformance.htm
 6) informationSource is missing PractitionerRole
 7) not sure why reasonCode has be changed from 0..* to 0..1
 */
-* effective[x] and medication[x] MS
+* subject 1..1 MS
+* subject only Reference(CancerPatient)
```

Comment:
If we're removing references to CancerPatient, then why even bother having a CancerPatient profile?